src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        String[] colors = {"green", "black", "yellow", "white", "purple", "blue", "pink"};
+        Random r = new Random();
+        return colors[r.nextInt(0, 6)];
```

Comment:
where Color - your enum
```suggestion
        Color[] colors = Color.values();
        return colors[random.nextInt(colors.length)].name();
```